utilities/src/linked_item.rs

```diff
@@ -8,9 +9,15 @@ pub struct LinkedItem<Item> {
 	pub next: Option<Item>,
 }
 
+impl<Item> Default for LinkedItem<Item> {
```

Comment:
can you derive Default?

Reply:
rustc requires `Value: core::default::Default` bound to use `Default::default` in LinkedList impl, might makes `Value` kind of restricted?

Reply:
ah ok